src/modules/user-select.js

```diff
@@ -0,0 +1,23 @@
+const applyRules = require("../util/applyRules");
+
+const defaultNames = {
+  us: "us"
```

Comment:
can you add
```
user-select: none; as us:n
user-select: auto; as us:au
user-select: text; as us:t
user-select: contain; as us:c
user-select: all; as us:a
```
so that is spec compliant and we can use whitelist if we don't need all of them

Reply:
Ah, but as per `emmet` specs its just `user-select:${none};` thats why I didn't add

Reply:
I have added these pre-defined configs.